src/UI/Components/Diagram/init.ts

```diff
@@ -314,7 +319,13 @@ export default function diagramInit(
       isMainInstance: boolean,
     ) => {
       appendInstance(paper, graph, instance, services, isMainInstance);
-
+      const savedInstancesCoordinates = localStorage.getItem(
```

Comment:
weren't we going to use the metadata endpoint for this? I believe the BE was ready too. See : [slides planning](https://docs.google.com/presentation/d/15X82_MMk13Qs5R8eUncrAZjeq-BW_yXobZdOaDmUT74/edit#slide=id.g23ae9aa45fc_1_10)